servers/rendering/shader_language.cpp

```diff
@@ -7144,13 +7146,13 @@ Error ShaderLanguage::_parse_block(BlockNode *p_block, const FunctionInfo &p_fun
 			}
 
 			if (n->get_datatype() != TYPE_BOOL) {
-				_set_error("Middle expression is expected to be boolean.");
+				_set_error(RTR("Middle expression is expected to be boolean."));
```

Comment:
```suggestion
				_set_error(RTR("The middle expression is expected to be boolean."));
```
Here we have a full sentence with a verb, so "The" is needed.